core/src/main/java/com/linecorp/armeria/client/retry/ExponentialBackoff.java

```diff
@@ -95,4 +95,33 @@ public String toString() {
                           .add("multiplier", multiplier)
                           .toString();
     }
+
+    public static Builder builder() {
+        return new Builder();
+    }
```

Comment:
Instead of adding a builder method to each implementation, should we add a builder method `Backoff` interface?
```java
interface Backoff {
   static ExpontionalBackoffBuilder builderForExpontional() {
      ...
   }
}
```
Note that `ExponentialBackoff` is a package-private class that users can't access.